latex-template/vXXX/Snakefile

```diff
@@ -0,0 +1,36 @@
+number = "XXX"
+
+rule all:
+    input:
+        f"build/v{number}.pdf"
+
+rule build:
```

Comment:
["Note that Snakemake automatically creates missing directories before jobs are executed."](https://snakemake.readthedocs.io/en/stable/tutorial/basics.html)

Reply:
We shouldn't show stuff that's not needed